### PR TITLE
Reviewing docs

### PR DESCRIPTION
**Pull request by dino-saurabh**
finished reviewing docs uptill the visit section.
1.visits.md
2. visit_type.md
3. visit_attribute_type.md
4.location.md
5.location_attribute_type.md
6.location_tag.md

### CODE REVIEW

File: source/includes/Visit/visits.md [hunk 1]
```diff
@@ -5,12 +5,11 @@
 * A Visit in OpenMRS represents precisely what it sounds like: a time when a patient is actively interacting with the 
 the healthcare system, typically at a location.
 
-* The metadata differentiating different types of visits is a Visit Type. Visit Types displayed in the user interface, 
-also, can be searched against.
+* The metadata differentiating different types of visits is a Visit Type. Visit Types displayed in the user interface, also can be searched against.
```
Comment (dino-saurabh) [inline source/includes/Visit/visits.md:8]: @Akayeshmantha  what does this line actually mean I dint get it completely ??

**Comment by dino-saurabh** [comment]
@Akayeshmantha  please review!!

File: source/includes/Visit/visits.md (hunk 1 above)
Comment (Akayeshmantha) [reply source/includes/Visit/visits.md:8]: Visit type is metadata which differentiates different types of visits.

**Review by Akayeshmantha — APPROVED** [review]
Great work @LuGO0

**Pull request merged**